pkgs/tools/networking/mpack/default.nix

```diff
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "0k590z96509k96zxmhv72gkwhrlf55jkmyqlzi72m61r7axhhh97";
   };
 
-  patches = [ ./build-fix.patch ./sendmail-via-execvp.diff ];
+  patches = [ ./build-fix.patch ./sendmail-via-execvp.diff ./CVE-2011-4919.patch ];
```

Comment:
```suggestion
  patches = [
    ./build-fix.patch
    ./sendmail-via-execvp.diff
    ./CVE-2011-4919.patch
  ];
```

Reply:
This feels pedantic, especially for a security patch. I'm planning on revamping the package sometime this week (think nixfmt-rfc-style + a move to by-name).

Reply:
I apologize, I didn't mean to be pedantic. you can mark this as resolved.